README.rst

```diff
@@ -124,18 +124,24 @@ sites)::
   ./manage.py lms collectstatic
   ./manage.py cms collectstatic
 
+Setup Studio SSO for Development::
+
+  ./manage.py lms manage_user studio_worker example@example.com --unusable-password
+  # For production system do not provide the id and sceret. They should be autogenerated.
```

Comment:
@feanil this comment is confusing -- do you mean that they should autogenerate them and then provide them here? or they will be set automatically by some other method?

Reply:
```suggestion
  # The command below creates the application that studio uses to login.  For convenience we set the `client-id` 
  # and `client-secret` here, however in a production system you should let those be auto-generated and look 
  # them up in the django-admin so that you can set the corresponding studio settings with less guessable 
  # settings.
  #
  # The relevant studio settings are `SOCIAL_AUTH_EDX_OAUTH2_KEY` and `SOCIAL_AUTH_EDX_OAUTH2_SECRET`
```

What do you think about something like this?

Reply:
it's still unclear to me @feanil . what autogenerates them? where are they to be found in django admin? once they are found in django admin, should this command be run with those values?